configs/FLYCOLORH743/config.h

```diff
@@ -0,0 +1,141 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H743
+
+#define BOARD_NAME        FLYCOLORH743
+#define MANUFACTURER_ID   FLCO
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP388
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+#define BEEPER_PIN           PE3
+#define MOTOR1_PIN           PA0   
+#define MOTOR2_PIN           PA1   
+#define MOTOR3_PIN           PA2    
+#define MOTOR4_PIN           PA3   
+#define MOTOR5_PIN           PB0   
+#define MOTOR6_PIN           PB1   
+#define MOTOR7_PIN           PD12  
+#define MOTOR8_PIN           PD13  
```

Comment:
See https://betaflight.com/docs/development/manufacturer/manufacturer-design-guidelines#32-resource-selection-considerations

Reply:
These timer/port resources were suggested as the FC is also designed for iNav and Ardupilot use which requires flexibility in allocating motor and servo outputs. As this is a H7, the extra DMA resource required for 8 bitbanged DShot outputs should not be an issue.